ex1/Tema3-V.4-Daniel Paraschiv.js

```diff
@@ -0,0 +1,38 @@
+document.addEventListener("DOMContentLoaded", () => {
+
+    //it worked only after i added the DOMcontentloader event and after I initialized both form and button and changed the button type from submit to click.
+    const form = document.getElementById("contact-form");
+    const button = document.getElementById("button");
+
+    button.addEventListener("click", (e) => {
+            
+        // this line is to prevent the form to load twice
+        e.preventDefault();
+        
+        // constants declaration : name, email, text, formData
+        const formData = new FormData(form);
+        const nume = formData.get("nume")
+        const email = formData.get("email")
+        const text = formData.get("text")
+        
+        // alert message with ok button
+        alert("\n" + "\n" + "Un email la adresa: " + email + " a fost trimis." + "\n" + "\n" + "Va multumesc !");
+
+        // print into the console the form information  
+        console.log("This is the information extracted from the user-form: ")
+        console.log(`Nume: ${nume}`);
+        console.log(`Email: ${email}`);
+        console.log(`Mesaj: ${text}`);
+
+        // empty the input cell boxes after the information was printed on the console
+        const inputs = form.querySelectorAll('input');
+                inputs.forEach(input => input.value = '');
```

Comment:
```suggestion
      inputs.forEach(input => input.value = '');
```
Se poate imbuntatit identarea: in acelasi rand cu "const"-ul de mai sus